packages/go_router/test/logging_test.dart

```diff
@@ -1,28 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-import 'package:flutter_test/flutter_test.dart';
-import 'package:go_router/src/logging.dart';
-import 'package:logging/logging.dart';
-
-void main() {
-  // Reset the logger before each test.
-  setUp(() {
```

Comment:
Instead of removing this, can you write a test to make sure setlogging does not remove listener?

Reply:
good idea, done.